src/citrine/resources/project.py

```diff
@@ -458,6 +583,8 @@ def remove_user(self, user_uid: Union[str, UUID]) -> bool:
             Returns ``True`` if user successfully removed
 
         """
+        if self.session._accounts_service_v3:
```

Comment:
this got a little longer than expected, since in some cases I want to check that we're not in v3 instead of checking for v3, but I think it was still worth it